CotEditor/Sources/FilterField.swift

```diff
@@ -174,6 +172,11 @@ private final class InnerFilterField: NSSearchField {
             self.validateImage()
         }
         
+        // Workaround to fix the issue that the text field does not update (via controlTextDidChange) when selecting an item from the recents menu.
+        if let coordinator = (self.delegate as? FilterField.Coordinator) {
```

Comment:
I feel this strategy is not good in that InnerFilterField knows the existence of FilterField.Coordinator.